eo-runtime/src/test/eo/org/eolang/rust-tests.eo

```diff
@@ -335,14 +336,11 @@
     use eo_env::eo_enum::EO;
     use eo_env::eo_enum::EO::{EOVertex, EOError};
 
-    pub fn foo(env: &mut EOEnv) -> EO {
-      let eobytes = env.find("Q.org.eolang.bytes");
-      let copy = env.copy(eobytes as u32).unwrap();
-      if env.put(copy.clone(), &[0x00, 0x1a, 0xEE]).is_none() {
-        EOError("put failed".to_string())
-      } else {
-        EOVertex(copy)
-      }
+    pub fn foo(env: &mut EOEnv) -> Option<EO> {
+      let eobytes = env.find("Q.org.eolang.bytes")?;
+      let copy = env.copy(eobytes)?;
+      env.put(copy.clone(), &[0x00, 0x1a, 0xEE])?;
```

Comment:
@levBagryansky Should there also be 4 spaces missing at the beginning of the line with the code?